### PR TITLE
Issue 2 view an entry

### PR DESCRIPTION
**Pull request by ayshahussain141**
code journal entries

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -2,25 +2,83 @@
...
+  data.nextEntryId++;
+  data.entries.unshift(object);
 
-  var entries = {};
-  entries.title = $form.elements.title.value;
-  entries.url = $form.elements.url.value;
-  entries.notes = $form.elements.notes.value;
-  entries.nextid = data.nextEntryId++;
-  data.entries.unshift(entries);
+  renderList(data.entries[0]);
```
Comment (BrettAlbright) [inline js/main.js]: you want to call the renderList function with the new object you just created above.

File: js/main.js [hunk 2]
```diff
@@ -2,25 +2,83 @@
...
   $form.reset();
   $image.src = 'images/placeholder-image-square.jpg';
-  event.preventDefault();
+
+});
+var $ul = document.querySelector('ul');
+function renderList(entry) {
```
Comment (BrettAlbright) [inline js/main.js:33]: This function should take in an entry, and return an `li` element. This function should not append anything to the DOM

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Hey Aysha, great work so far! there are a couple updates noted in the comments. Also, you need to insure that when the page is refreshed, the same view is showing as before the refresh. Reach out with any questions you have on that!

**Comment by ayshahussain141** [comment]
code journal entries 2 0

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**